web_src/js/components/ScopedAccessTokenSelector.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <div class="scoped-access-token-category">
+    <div class="field gt-pl-2">
+      <label class="checkbox-label">
+        <input
+          ref="category"
+          v-model="categorySelected"
+          class="scope-checkbox scoped-access-token-input"
+          type="checkbox"
+          name="scope"
+          :value="'delete:' + category"
+          @change="onCategoryInput"
```

Comment:
Hmm… Interesting.
I just tested it, and surprisingly it works with <kbd>Space</kbd>.
However, it doesn't work if you set it manually from the console.
As I've had to learn myself, it is probably better to use
```suggestion
          @input="onCategoryInput"
```